definitions/procedures/backup/compress_data.rb

```diff
@@ -0,0 +1,26 @@
+module Procedures::Backup
+  class CompressData < ForemanMaintain::Procedure
+    metadata do
+      description 'Compress backup data to save space'
+      tags :backup
+      param :backup_dir, 'Directory where to backup to', :required => true
+    end
+
+    def run
+      pg_tar = File.join(@backup_dir, 'pgsql_data.tar')
+      if File.exist?(pg_tar)
```

Comment:
Apart from tar file name & message inside spinner, rest of the things are same for both. 
Can we extract this logic into a separate method or loop over these two values?